packages/next/src/client/components/react-dev-overlay/ui/styles/base.tsx

```diff
@@ -33,6 +33,7 @@ export function Base({ scale = 1 }: { scale?: DevToolsScale }) {
             'Source Sans Pro', sans-serif;
 
           font-family: var(--font-stack-sans);
+          font-variant-ligatures: none;
```

Comment:
Easy enough. Does this mean ligatures are still bundled in Geist Mono? Font size isn't that big of a deal in this case but still surprised these are always bundled.